metron-platform/metron-data-management/src/main/java/org/apache/metron/dataloads/nonbulk/taxii/TaxiiLoader.java

```diff
@@ -165,6 +167,19 @@ public static Options getOptions() {
   public static final long DEFAULT_TIME_BETWEEN_POLLS = ONE_HR_IN_MS;
 
 
+  public static boolean isStixExtractor(Extractor e) {
+    if(e instanceof StixExtractor) {
+      return true;
+    }
+    else if(e instanceof ExtractorDecorator) {
+      ExtractorDecorator decorator = (ExtractorDecorator)e;
+      if(decorator.getUnderlyingExtractor() != null && decorator.getUnderlyingExtractor() instanceof StixExtractor) {
+        return true;
+      }
+    }
+    return false;
+  }
+
```

Comment:
why not just?
`e instanceof StixExtractor || (ExtractorDecorator)e.getUnderlyingExtractor() instanceof StixExtractor`
this is java after all and has shortcutting... even stellar does that now too :)